### DISCUSSION TITLE
The parent austenite grain reconstruction destroyed the microstructure near the crack

### DISCUSSION THREAD

**Discussion by wanghaijian217** [ORTools4MTEX/ORTools#48] [Q&A] [answered]
Hi Frank and Azdiar
Recently, we tried to observe the original austenite grain near the crack, and found that the fracture surface was reconstructed after reconstruction, but the structure at the fracture surface was not displayed。
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/c72aaf1c-f0ec-4ff8-b9c3-9a66b19b872b)
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/33911c58-189f-43fd-8f0d-4c0bf1867d4c)
The structure near the crack has been reconstructed, which makes it impossible for me to analyze the crack. How can this be solved?
best wishes
Long

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @wanghaijian217,

I hope you take the following few points in the right spirit and as a learning experience.
Your answer is at the bottom of this message.

While Frank and I are always willing to help out, please remember that we are not working on your project or know the precise details related to your samples and maps. Therefore, in order for us to help you:

1. Please provide ALL the background information related to the sample and map that could help us understand what it is that we are looking at.  

2. There is a lot of online MTEX and ORTools documentation that you could help you answer your questions through a simple Google search. So please spend some time reading and learning how to effectively and correctly use the various functions created in MTEX and ORTools first before applying them.

3.  Rather than being in a hurry to post something quickly as a question, please spend some time formulating a coherent question. For example, the sentence "...fracture surface was reconstructed after reconstruction, but the structure at the fracture surface was not displayed."  is unclear and requires us to interpret exactly what you are asking of us.

4. Finally, once we have answered the questions contained in your many emails and posts to us, please provide feedback by marking the answer that helped you out the most in the discussion forum. A posted acknowledgment for the help by way of a little "thank you" also goes a long way too! :) 



Now, to answer your question:
I assume you are saying that after loading a map into MTEX or ORTools, the child grains are getting filled in upon initial grain reconstruction. Please have a look at the following [webpage](https://mtex-toolbox.github.io/GrainReconstructionAdvanced.html). I believe Option 3 may be of most relevance to your case.

Also, here is a [weblink to a paper](https://onlinelibrary.wiley.com/doi/epdf/10.1002/adem.202200891) that may be of relevance to you. It specifically deals with reconstructing PAGs at voids and cracks using ORTools.

Warm regards,
Azdi

**Comment by wanghaijian217** [comment]
Best wishes to Frank and Azdiar
Thank you very much for your advice, your help in providing a new approach to refactoring practice, thank you very much.
